contrib/storage-http/src/test/resources/data/sample_data.json

```diff
@@ -0,0 +1,1000 @@
+[{"id":1,"first_name":"Eba","last_name":"Le Sarr","email":"elesarr0@springer.com","gender":"Female","ip_address":"249.113.19.117","random_number":92.3},
```

Comment:
We don't need such big file for tests, please make it as small as possible.

Reply:
I removed the file and the one unit test that used it.